### PR TITLE
[#95] Implement wrappers for most popular popular actions

### PR DESCRIPTION
**Pull request by jmfayard**
See https://codecov.wordpress.com/2020/09/14/most-popular-and-used-github-actions/

### CODE REVIEW

**Comment by jmfayard** [comment]
Update, I now automatically and syggest the relevant `IntegerTyping` and `BooleanTyping`

```
Generating gradle/gradle-build-action@v2...
ActionCoords("gradle", "gradle-build-action", "v2") I suggest the following typings:
    mapOf(
            "cache-write-only" to BooleanTyping,
            "gradle-home-cache-strict-match" to BooleanTyping,
    )    
```

File: library/src/gen/kotlin/it/krzeminski/githubactions/actions/anmol098/WakaReadmeStatsV4.kt [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ * Are you an early 🐤 or a night 🦉? When are you most productive during the day? Let's check out
+ * in your readme!
+ *
+ * [Action on GitHub](https://github.com/anmol098/waka-readme-stats)
+ */
+public class WakaReadmeStatsV4(
+    /**
+     * GitHub access token with Repo scope
+     */
+    public val gHTOKEN: String? = null,
```
Comment (jmfayard) [inline library/src/gen/kotlin/it/krzeminski/githubactions/actions/anmol098/WakaReadmeStatsV4.kt:22]: There is a bug here: the property is `GH_TOKEN`
https://github.com/anmol098/waka-readme-stats/blob/master/action.yml

Comment (jmfayard) [reply library/src/gen/kotlin/it/krzeminski/githubactions/actions/anmol098/WakaReadmeStatsV4.kt]: Solved

**Comment by jmfayard** [comment]
@krzema12 
Any idea why https://raw.githubusercontent.com/pre-commit/action/v2/action.yml doesn't work?
There is a V2.0.0 release: https://github.com/pre-commit/action/releases/tag/v2.0.0
Should we use https://raw.githubusercontent.com/pre-commit/action/master/action.yml instead?

It's not only this repository, there are like 30 who are not working, see `ErrorWrapperRequest` in `WrappersToGenerate.kt`

**Comment by krzema12** [comment]
@jmfayard see https://raw.githubusercontent.com/pre-commit/action/v2.0.0/action.yml

`v2` != `v2.0.0` :) If a given action wants to be used with `v2` version, it needs to have this specific tag: `v2`. This action doesn't have one: https://github.com/pre-commit/action/tags

**Comment by krzema12** [comment]
While I appreciate the contribution, this PR is way too big - changes multiple things at once.

I see you adjust the generator and it influences existing wrappers. Technically it's a breaking change and should be separated from such high-volume changes. So before we add any new wrappers, if you want to make any adjustments to the generator (both that influence or don't influence existing generated wrappers), let's make these adjustments in a separate PR(s). The smaller PR the better, it's easier to review. For example, I see you change types in some existing wrappers - would be good to have a separate PR for it so that me as a reviewer understands why. The very last PR should contain only adding generated wrappers, without any other changes.

**Comment by krzema12** [comment]
BTW, I don't see the commits coming from automatic wrapper regenerating job. Could you check your fork's actions if they fail because of some reason?

**Comment by jmfayard** [comment]
Convert to draft because it depends on https://github.com/krzema12/github-actions-kotlin-dsl/pull/102

**Pull request closed without merging**